pkg/ccl/changefeedccl/resolvedspan/frontier.go

```diff
@@ -15,7 +14,6 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
 	"github.com/cockroachdb/cockroach/pkg/settings"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
-	"github.com/cockroachdb/cockroach/pkg/util/log"
```

Comment:
The original PR also unfortunately missed deleting some code in the second commit which prevents it from building, it's been corrected in this backport